libs/designsystem/slide/src/slides.component.scss

```diff
@@ -77,6 +77,11 @@ $bullet-inactive-opacity: 0.2;
     margin-block: -(utils.size('m'));
   }
 
+  // Allow for swiper content to have equal height.
+  swiper-slide {
+    height: auto;
+  }
```

Comment:
If we go with the attribute directive approach it would be good to scope this to only apply in that situation:

 ```
 swiper-slide:has([slideStretch]) {
    height: auto;
  }
```

Reply:
The `height: auto` css property on swiper-slide has no implication on the current slider, unless you define 100% height on children. Theres no need for configuration.